### PR TITLE
Update to version 2.0.2

### PR DESCRIPTION
**Pull request by jjhelmus**
Update recipe to version 2.0.2 with new requirements.
Drop Python 2.x support as the upstream project is Python 3 only.
Sync with AnacondaRecipes

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,36 +13,42 @@ source:
...
 
 requirements:
-  build:
+  host:
     - python
     - pip
+    - pytest-runner
   run:
     - python
-    - setuptools
```
Comment (jakirkham) [inline recipe/meta.yaml]: Think we need this as they use [`pkg_resources` a few places]( https://github.com/PyCQA/astroid/search?p=1&q=pkg_resources&unscoped_q=pkg_resources ). Admittedly most of those are tests, but a few are not.

**Comment by jakirkham** [comment]
Thanks @jjhelmus. This looks basically fine. Had one comment above.

**Comment by jjhelmus** [comment]
`setuptools` was added back as a run requirement.

**Comment by jakirkham** [comment]
Thanks @jjhelmus 😄

**Pull request merged**